### DISCUSSION TITLE
Crashing Constantly

### DISCUSSION THREAD

**Discussion by GreatBlueMoon** [FakeFishGames/Barotrauma#14483] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Playing the game simply causes the crash randomly.

### Reproduction steps

[crashreport (24).log](https://github.com/user-attachments/files/16733474/crashreport.24.log)

### Bug prevalence

Happens everyday i play.

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server) / and tested dedicated server via new crash reports

### Version

v1.6.101.0 (unstable) (and current version now)

### Which operating system did you encounter this bug on?

Windows

**Comment by GreatBlueMoon** [comment]
Unrelated but heres other crash reports over the recent time i've played normal games and other such as well for the regular crashing.
[crashreport (22).log](https://github.com/user-attachments/files/16733482/crashreport.22.log)
[crashreport (23).log](https://github.com/user-attachments/files/16733483/crashreport.23.log)

**Comment by GreatBlueMoon** [comment]
[servercrashreport (19).log](https://github.com/user-attachments/files/16733485/servercrashreport.19.log)
[servercrashreport (18).log](https://github.com/user-attachments/files/16733486/servercrashreport.18.log)
[servercrashreport (17).log](https://github.com/user-attachments/files/16733493/servercrashreport.17.log)
[servercrashreportslim.log](https://github.com/user-attachments/files/16733495/servercrashreportslim.log)

**Comment by GreatBlueMoon** [comment]
[servercrashreport (24).log](https://github.com/user-attachments/files/16738525/servercrashreport.24.log)
[servercrashreport (23).log](https://github.com/user-attachments/files/16738526/servercrashreport.23.log)
[servercrashreport (22).log](https://github.com/user-attachments/files/16738527/servercrashreport.22.log)
[servercrashreport (21).log](https://github.com/user-attachments/files/16738528/servercrashreport.21.log)
[servercrashreport (20).log](https://github.com/user-attachments/files/16738529/servercrashreport.20.log)
keep crashing Normal gameplay dedicated server this time, desperate for help

**Comment by Regalis11** [comment]
Sorry for taking so long to get back to you! There seem to be several different kinds of crashes here. The ones that mention "AIObjectiveDeconstructItems.IsValidTarget" should've been fixed in the latest update. Some seem to be instances of https://github.com/FakeFishGames/Barotrauma/issues/10540, which can sometimes be an issue at the user's end: there are some potential workarounds in the linked ticket.

The one in the initial post seems like something I haven't encountered before though. Have you noticed it happening in some specific situation? The crash report seems to indicate that multiple SMG magazines were somehow placed into a slot that they shouldn't be possible to place into.

In any case, I'm sorry for the inconvenience! I hope the recent hotfix takes care of most of the issues you've been encountering.
